package.json

```diff
@@ -10,16 +10,20 @@
     "src"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=14"
```

Comment:
can't run on test on node 12 because `tsup` depends on `bundle-require` which uses nullish coalescing which isn't supported in node 12. since we can't test on 12 I don't want to say it works. node 12 is nearing EOL so may be fine never supporting it.